services/sensiolabs/sensiolabs.tester.js

```diff
@@ -0,0 +1,176 @@
+'use strict'
+
+const Joi = require('joi')
+const { colorScheme } = require('../test-helpers')
+const t = (module.exports = require('../create-service-tester')())
+const { withRegex } = require('../test-validators')
+
+const {
+  runningMockResponse,
+  platinumMockResponse,
+  goldMockResponse,
+  silverMockResponse,
+  bronzeMockResponse,
+  noMedalMockResponse,
+  mockSLUser,
+  mockSLApiToken,
+  mockSensiolabsCreds,
+  restore,
+  tokenExists,
+  logTokenWarning,
+} = require('./sensiolabs-test-helpers')
+
+const gradeRegex = /platinum|gold|silver|bronze|no medal/
+const sampleProjectUuid = '45afb680-d4e6-4e66-93ea-bcfa79eb8a87'
+
+t.create('live: valid project')
+  .before(logTokenWarning)
+  .get(`/${sampleProjectUuid}.json`)
+  .timeout(10000)
```

Comment:
I've been running some tests locally (using my own user/api token that I created) to validate this test in "live" mode. More often than not it is timing out, even with this 10s timeout period. Hopefully it's just the airport wifi being slow though

Reply:
This is working for me better now (even on hotel wifi) so I'm going to chalk it up to the airport network. Will keep an eye on this test once merged though to see if timeouts are a problem